utils/scale-test

```diff
@@ -154,6 +199,9 @@ def main():
         '--multi-file', action='store_true',
         default=False, help='vary number of input files as well')
     parser.add_argument(
+        '--sum-multi', action='store_true',
+        default=False, help='simulate a multi-primary run and sum stats')
```

Comment:
When swiftc is invoked with N files, say `swiftc A.swift B.swift C.swift`, its driver turns around and runs `swiftc -frontend --primary-file A.swift A.swift B.swift C.swift`, `swiftc -frontend --primary-file B.swift A.swift B.swift C.swift`, `swiftc -frontend --primary-file C.swift A.swift B.swift C.swift`

We _hope_ that in cases like this, it only does quadratic amounts of work parsing the files N*N times and then typechecks-and-translates each file once (=N units of work) but in some cases it does more. `--sum-multi` simulates such a run, and sums the statistics over all the sub-frontend-jobs.